cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AliasesTest.scala

```diff
@@ -0,0 +1,318 @@
+      query(
+        """ALTER ALIAS `remote-with-driver-settings` SET DATABASE USER bob PASSWORD 'newPassword'
+           |DRIVER { connection_timeout: duration({ minutes: 1 }), logging_level: "debug" }""".stripMargin, ResultAssertions(r => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+        note{
+          p(
+            """All driver settings are replaced by the new ones.
+              |In this case, by not repeating the driver setting `connection_pool_max_size` the value will be deleted and fallback to the default value.""".stripMargin)
+        }
+      }
+      p("Example of altering a remote alias to remove all custom driver settings.")
+      query("ALTER ALIAS `movie scripts` SET DATABASE DRIVER { }", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+      }
+      p("When a local database alias has been altered, it will show up in the aliases column for the target database provided by the command `SHOW DATABASES`.")
+      query("SHOW DATABASE `northwind`", assertNameField("northwind-graph-2021")) {
+        resultTable()
+      }
+      p("The changes for all database aliases will show up in the `SHOW ALIASES FOR DATABASE` command.")
+      query("SHOW ALIASES FOR DATABASE YIELD * WHERE name in ['northwind', 'remote-northwind', 'remote-with-driver-settings', 'movie scripts']",
+        assertNameField("northwind", "remote-northwind", "remote-with-driver-settings", "movie scripts")) {
+        resultTable()
+      }
+      p("""This command is optionally idempotent, with the default behavior to fail with an error if the alias does not exist.
+        |Appending `IF EXISTS` to the command ensures that no error is returned and nothing happens should the alias not exist.""".stripMargin)
+      query("ALTER ALIAS `no-alias` IF EXISTS SET DATABASE TARGET `northwind-graph-2021`", ResultAssertions(r => {
+        assertStats(r)
+      })) {
+        statsOnlyResultTable()
+      }
+    }
+    section(title="Deleting database aliases", id="alias-management-drop-database-alias", role = "enterprise-edition") {
+      p(
+        """Both local and remote aliases can be deleted using the `DROP ALIAS` command.
+          |The required privileges are described <<cypher-manual#access-control-dbms-administration-alias-management, here>>.""".stripMargin)
+      query("DROP ALIAS `northwind` FOR DATABASE", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+      }
+      query("DROP ALIAS `remote-northwind` FOR DATABASE", ResultAssertions(r => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+      }
+      p("When a database alias has been deleted, it will no longer show up in the aliases column provided by the command `SHOW DATABASES` or in the `SHOW ALIAS FOR DATABASE` command.")
```

Comment:
```suggestion
      p("When a database alias has been deleted, it will no longer show up in the aliases column provided by the command `SHOW DATABASES` or in the `SHOW ALIASES FOR DATABASE` command.")
```
Cherry pick of the change made in the 4.4 branch via #1545